test/extensions/secondorder/diag_ggn/diaggnn_settings.py

```diff
@@ -7,20 +7,23 @@
 from test.extensions.secondorder.secondorder_settings import SECONDORDER_SETTINGS
 from torch.nn import ELU, SELU
 
-from test.extensions.automated_settings import make_simple_cnn_setting
+from test.extensions.automated_settings import make_simple_act_setting
 
 DiagGGN_SETTINGS = []
 
 SHARED_SETTINGS = SECONDORDER_SETTINGS
+
+
 LOCAL_SETTINGS = []
 
+
 ###############################################################################
 #                         test setting: Activation Layers                     #
 ###############################################################################
 activations = [ELU, SELU]
```

Comment:
no `LogSigmoid` here?

Reply:
No, we have it in `secondorder_settings.py`

Reply:
https://github.com/f-dangel/backpack/blob/f362a4c67551f033d5a8be208d8566421bca36ac/test/extensions/secondorder/secondorder_settings.py#L95 ; here